components/LabelText/LabelText.test.tsx

```diff
@@ -0,0 +1,25 @@
+import { LabelText } from './LabelText';
+import { render } from '@testing-library/react';
+import React from 'react';
+
+describe('LabelText Component', () => {
+  it('renders properly', () => {
+    const { getByTestId } = render(<LabelText>Test Label</LabelText>);
```

Comment:
nit: this feels a bit cluttered here, do you think it would look better like this:

```
render(
  <LabelText>
      Test Label
  </LabelText>
);
```

Reply:
This is how it is being formatted. If I change it, it'll just change it back.

Reply:
Confirmed I cannot change the formatting of this.